fw/http_sess_conf.c

```diff
@@ -371,10 +380,22 @@ tfw_cfgop_cookie_options_set(TfwCfgSpec *cs, TfwCfgEntry *ce)
 
 	TFW_CFG_ENTRY_FOR_EACH_ATTR(ce, i, key, val) {
 		if (!strcasecmp(key, "Path")) {
+			if (was_path) {
+				T_ERR_NL("Duplicate argument: '%s'\n", key);
+				return -EINVAL;
+			}
```

Comment:
This repetitive code pattern clearly asks for a macro that includes the check **and** the assignment to `true`. There are places where the assignment is done after additional piece of code such as parsing of the value. But if that parsing fails it won't matter anyway, so the assignment to `true` can be done earlier in the macro.

Reply:
Yes good point! Fixed